src/digital_identity_service/index.js

```diff
@@ -9,6 +13,45 @@ class DigitalIdentityService {
     this.pem = pem;
     this.apiUrl = apiUrl;
   }
+
+  /**
+   *
+   * @param shareSessionConfig
+   * @returns {Promise<ShareSessionResult>}
+   */
+  createSession(shareSessionConfig) {
+    const jsonShareSession = JSON.parse(JSON.stringify(shareSessionConfig));
```

Comment:
I think we also need to add some builders for the session config. I had done that in the draft PR, `ShareSessionBuilder` and `ShareSessionNotificationBuilder`. please check those.